static/app/views/preprod/sizeAnalysis.tsx

```diff
@@ -0,0 +1,23 @@
+import {FeatureBadge} from 'sentry/components/core/badge/featureBadge';
+import * as Layout from 'sentry/components/layouts/thirds';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {t} from 'sentry/locale';
+
+function SizeAnalysis() {
+  return (
+    <SentryDocumentTitle title={t('Size analysis')}>
+      <Layout.Header unified>
+        <Layout.HeaderContent>
+          <Layout.Title>
+            Size analysis <FeatureBadge type="new" />
```

Comment:
We unfortunately don't lint for translations
```suggestion
            {t('Size analysis')} <FeatureBadge type="new" />
```